vagrant_box_defaults.rb

```diff
@@ -6,3 +6,5 @@
 $SERVER_VERSION= "167"
 $NETNEXT_SERVER_BOX= "cilium/ubuntu-next"
 $NETNEXT_SERVER_VERSION= "45"
+@v419_SERVER_BOX= "cilium/ubuntu-4-19"
```

Comment:
Where is this one defined?

Reply:
This file is the source of truth about which box/version combo should be used for given branch. This image has been introduced by this PR: https://github.com/cilium/packer-ci-build/pull/197 (and two subsequent fixes)